packages/console/src/components/CustomUiAssetsUploader/index.tsx

```diff
@@ -13,6 +13,8 @@ import FileIcon from '../FileIcon';
 
 import * as styles from './index.module.scss';
 
+const requestTimeout = 300_000; // 5 minutes
```

Comment:
does the api have the same timeout?

Reply:
Good point. I didn't set it on the server side, will update

Reply:
Just confirmed we have no timeout set on the API side, so a [new task](https://linear.app/silverhand/issue/LOG-9693/set-1-min-timeout-to-all-apis) has been added and we'll merge this PR first.